containerz/containerz.proto

```diff
@@ -118,6 +119,23 @@ service Containerz {
 
   // ListVolume returns all volumes that match the spec defined in the filter.
   rpc ListVolume(ListVolumeRequest) returns (stream ListVolumeResponse) {}
+
+  // StartPlugin starts the specified plugin identified by the name field
+  // of the request. It should be started using the instance_name of the
+  // request. If the plugin file named `name.tar` does not exist, this
```

Comment:
I presume the plugin needs to be a packaged OCI image right? Does this necessarily mean it must be `name.tar`?

Reply:
Yes - fair point. Basically we need to make sure that the file uploaded during deploy is referenceable when PluginStart is called. So there is no need to mandate an extension.